bitrise.yml

```diff
@@ -208,7 +208,7 @@ workflows:
         inputs:
         - webhook_url: $WEBHOOK_SLACK_CUX_BOTS
         - webhook_url_on_error: $WEBHOOK_SLACK_CUX_BOTS
-        - text_on_error: "iOS build failed (c.c. @connections-consumer-bank-network-oncall @mats)"
+        - text_on_error: "iOS build failed (c.c. @link-mobile-run @mats)"
```

Comment:
Should we remove Mat to reduce the load?

Reply:
@mats-stripe wants to be there 😅

Reply:
🤷‍♂️

Reply:
https://github.com/stripe/stripe-ios/pull/4811/commits/60229ae1982a5fa6b76e3c180ff472888e0b0fea